apps/sensenet/src/store/ActionHelpers.ts

```diff
@@ -0,0 +1,12 @@
+export const createAction = <TReturns extends { type: string }, TArgs extends any[]>(
```

Comment:
This file is not needed anymore. You can import these helpers from sn-redux.

Reply:
👍